test/validators.js

```diff
@@ -4629,6 +4629,12 @@ describe('Validators', () => {
       valid: [''],
       invalid: ['a', 'ab'],
     });
+    test({
```

Comment:
I think we can remove this new test since it does not test the new functionality

Reply:
Yess!! Removed that test.